src/Serilog.Sinks.NLog/Sinks/NLog/NLogSink.cs

```diff
@@ -29,34 +29,45 @@ public void Emit(LogEvent logEvent)
                 }
             }
 
+            var level = GetMappedLevel(logEvent);
             var message = logEvent.RenderMessage(formatProvider);
             var exception = logEvent.Exception;
 
+            var nlogEvent = new LogEventInfo(level, loggerName, message)
+            {
+                Exception = exception
+            };
+
+            // pass along the event's properties to nlog
+            const string literalFormat = "l";
```

Comment:
`l` is only valid when the property value is a `ScalarValue` with inner `Value` of type `string` - but otherwise, this seems sane to me :+1: